src/Microsoft.Data.SqlClient/tests/ManualTests/DataCommon/SqlClientCustomTokenCredential.cs

```diff
@@ -95,7 +98,7 @@ public static async Task<AccessToken> AzureActiveDirectoryAuthenticationCallback
         {
             using CancellationTokenSource cts = new();
             cts.CancelAfter(30000); // Hard coded for tests
-            string[] scopes = new string[] { resource };
+            string[] scopes = new string[] { resource + "/.default" };
```

Comment:
NIT: I think it would be better if we define a constant value for this "/.default" string for better readability and preventing runtime overhead.

Reply:
Done